packages/manager/modules/emailpro/src/domain/dkim-autoconfig/emailpro-domain-dkim-autoconfig.html

```diff
@@ -4,6 +4,7 @@
         data-wizard
         data-wizard-on-cancel="resetAction"
         data-wizard-on-finish="ctrl.onFinishDkim.bind(ctrl)"
+        data-wizard-cancel-button-text="ctrl.hideConfirmButton ? 'emailpro_tab_domain_diagnostic_dkim_cancel' : 'emailpro_tab_domain_diagnostic_dkim_close' | translate"
```

Comment:
Don't you need to use () around the ternary operation ? like this :
```suggestion
        data-wizard-cancel-button-text="(ctrl.hideConfirmButton ? 'emailpro_tab_domain_diagnostic_dkim_cancel' : 'emailpro_tab_domain_diagnostic_dkim_close') | translate"
```
Because with your proposition, I am scare only the last content is translated

Reply:
I tested it and it worked. 

I am going to check it again to make sure there is no issues.

Reply:
Cf https://github.com/prettier/prettier/issues/5586 for some explanation